### PR TITLE
Update runtime to KDE 5.12

### PR DESCRIPTION
**Pull request by FakeShemp**

### CODE REVIEW

File: org.kde.katomic.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     "id": "org.kde.katomic",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.11",
+    "runtime-version": "5.12",
     "sdk": "org.kde.Sdk",
     "command": "katomic",
     "rename-icon": "katomic",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
```
Comment (tsdgeos) [inline org.kde.katomic.json]: Can you please explain in the commit log this change? All the others changes are either explained or self explanatory but this one is a bit different

Comment (FakeShemp) [reply org.kde.katomic.json]: `--fallback-x11` means that Flatpak will try to launch with Wayland firstly and only fall back on x11 if Wayland didn't work out all right.

Comment (tsdgeos) [reply org.kde.katomic.json]: Sounds good, can you please update the commit log to say that?

Comment (FakeShemp) [reply org.kde.katomic.json]: Done :)

**Comment by nedrichards** [comment]
If you use fallback-x11 then you should probably set the minimum version to 0.10.2, as otherwise versions before that will crash when they see an unknown finish-arg. Not a big deal as there isn't much <1.0 in our usage these days but it's still the default in e.g. debian stable without backports.

**Comment by FakeShemp** [comment]
It failed at upload, so build should still be good.

**Pull request merged**